app/views/govuk_publishing_components/components/docs/big_number.yml

```diff
@@ -0,0 +1,52 @@
+name: Big number
+description: A big number, with a label if needed, for visualising quantitative values such as the number of government departments or historical prime ministers.
+body: |
+  This component requires at least a non-falsey `number` attribute value passed to it, otherwise it will not render.
+accessibility_criteria: |
+  This component must:
+  - uses clear language and is positioned in a logical context on the page it's being used on
+  - communicates number value and label (if present) in a single dictation when reading with screen reader software
```

Comment:
Nitpick on language here. It currently reads a little strangely.

```
This component must uses clear language and is positioned in a logical context
This component must communicates number value and label
```

Also I think accessibility criteria should define the way the component itself behaves, not how it should be used - effectively it should be a set of rules that the component must adhere to in order to be accessible. Something like:

```
The component must:

- communicate number value and label (if present) in a single dictation when read with a screen reader
- convey the meaning of the number shown
```

Reply:
Your suggestion makes sense to me, updating...